msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1372,6 +1372,21 @@ turnContext, CancellationToken cancellationToken) {
 ```
 * * *
 
+## Uninstall event
+
+After the post uninstall behavior for bots in the personal scope with the Teams and groupChat scopes are aligned, you cannot send or receive messages after an app has been uninstalled. Your bot receives a 403 response code to new messages posted by your bot. The 403 response code has fields that give the reason behind for the 403 response, which is either the app was uninstalled or the bot was blocked.
+
+![Uninstall event](~/assets/images/bots/uninstallbot.png)
+
+## Event handling
+
+As you begin using these new install and uninstall events, there are some instances where bots throw exceptions on receiving unexpected events from Teams. Some of the reasons why this occurs:
+
+* You have built your bot without the Microsoft Bot Framework SDK, and the bot throws an exception on receiving an unexpected event.
+* You built your bot with the Microsoft Bot Framework SDK, and you chose to alter the default event behavior by overriding the base event handle.
+
+It is important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you are using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you do not choose to handle.
```

Comment:
This section seems to be talking about install events rather than uninstall events. Are there any code examples that we are looking to add here?

Reply:
This section is about event handling not uninstalling of bots. Ojasvi specifically asked to add this section. There are no code samples.